lambda/lexv2-build/Makefile

```diff
@@ -1,6 +1,6 @@
 NAME=$(shell basename $(shell pwd))
 DST=../../build/lambda/$(NAME).zip
-
+RESOURCES := $(shell  find . | grep -v test )
 
 $(DST): $(RESOURCES) 
-	echo "Building $(NAME)"; pip3 install -r requirements.txt -t . && zip -r -q $(DST) .
+	echo "Building $(NAME)"; pip3 install -r requirements.txt -t ./py_modules && zip -r -q $(DST) .
```

Comment:
This also seems unrelated to PR title - "Kendra webcrawler fixes"  - probably should have been in a separate PR for clarity, but doesn't really matter.

Reply:
sorry -- yes. i should have created a separate PR for this..